src/Storages/System/StorageSystemDashboards.cpp

```diff
@@ -220,9 +220,10 @@ ORDER BY t WITH FILL STEP {rounding:UInt32}
             { "title", "Concurrent network connections" },
             { "query", trim(R"EOQ(
 SELECT toStartOfInterval(event_time, INTERVAL {rounding:UInt32} SECOND)::INT AS t,
-    sum(CurrentMetric_TCPConnection) AS TCP_Connections,
-    sum(CurrentMetric_MySQLConnection) AS MySQL_Connections,
-    sum(CurrentMetric_HTTPConnection) AS HTTP_Connections
+    avg(CurrentMetric_TCPConnection) AS TCP_Connections,
```

Comment:
I want total connections on all servers in the cluster.
It should take the sum across servers with grouping by seconds, then take the maximum over the rounding time interval.